src/Interpreters/QueryMetricLog.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include <Common/ProfileEvents.h>
+#include <Common/CurrentMetrics.h>
+#include "Core/BackgroundSchedulePool.h"
```

Comment:
```suggestion
#include <Core/BackgroundSchedulePool.h>
```

Reply:
Yep yep, that's VSCode adding headers automatically. I missed this one. I can't see how to configure it to use <> instead of double quotes.